index.html

```diff
@@ -23,7 +24,17 @@
             </div>
             <nav class="menu">
```

Comment:
[OPTIONAL] I think you can make the navbar's design in the mobile version match the Figma template design better. I'm seeing a wider margin with the top of the screen than the one present in the template design. Consider fixing it.